src/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Response, Headers } from 'node-fetch';
+import mapKeys from 'lodash.mapkeys';
+import identity from 'lodash.identity';
+import FormData from 'form-data';
+import { AxiosInstance, AxiosRequestConfig } from './types';
+
+export interface FetchInit extends Record<string, any> {
+  headers?: Record<string, string>;
+  method?: AxiosRequestConfig['method'];
+  body?: FormData | any;
+  extra?: any;
+}
+
+type AxiosTransformer = (config: AxiosRequestConfig, input: string | undefined, init: FetchInit) => AxiosRequestConfig;
+
+type AxiosFetch = (input?: string, init?: FetchInit) => Response;
```

Comment:
Will a consumer want to import this type? Is it worth exporting just in case?

Reply:
Exported